portifolio.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+    <header>
+        <nav class="logo">
+           <a href="#"><h3>KGW</h3></a>
+            <ul>
+                <li><img src="image/Union.png" alt="bars"></li>
+                <li></li>    
+            </ul>            
+        </nav>
```

Comment:
Nice job using the `header` tag 👍 💯 . Kindly, move the `logo(i.e. KGW` from `nav`. The `nav` tag should have to include only the right side elements which mean `hamburger button` for mobile view and in near future `ul` elements for desktop view. Use `flexbox` for the `header` tag to style your elements inside.